File: Backend-2nf-Clothing-Web/2nf-API/Requests/ClientRequest.cs
namespace _2nf_API.Requests
{
    public class ClientRequest
    {
        public string Name { get; set; }
        public string? SecondName { get; set; }
        public string Surname { get; set; }
        public string? SecondSurname { get; set; }
        public int DocId { get; set; }
        public string Street { get; set; }
        public int StreetNumber { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string Country { get; set; }
        public string Phone { get; set; }
        public string PostalCode { get; set; }
        public string Email { get; set; }
        public string UserName { get; set; }
        public string Password { get; set; }
        public string? NewPassword { get; set; }
    }
}

File: Frontend-2nf-Clothing-Web/src/app/models/clients/clientRequest.ts
export interface ClientRequest {
    name: string;
    secondName?: string;
    surname: string;
    secondSurname?: string;
    docId: number;
    street: string;
    streetNumber: number;
    city: string;
    state: string;
    country: string;
    postalCode: string;
    phone: string;
    email: string;
    username: string;
    password: string;
    newPassword?: string;
}